src/spikeinterface/core/node_pipeline.py

```diff
@@ -366,13 +365,14 @@ def run_node_pipeline(
 
     init_args = (recording, nodes)
 
+    job_kwargs["job_name"] = "pipeline"
```

Comment:
I am not sure we want to modify in place a global job_kwargs dict we have in many script.

Reply:
This does not modify the global job_kwargs here. It is just replicating the behavior that we had before. 

Or maybe I am missing something?